components/ui/data-table/data-table.tsx

```diff
@@ -86,6 +86,9 @@ export function DataTable<TData extends Record<string, unknown>, TValue>(props:
   const table = useReactTable({
     data: props.data,
     columns: props.columns,
+    defaultColumn: {
+      size: 0
```

Comment:
Default is 150 which I knew but forgot for line 31 which was causing every column to have a width of 150 instead of being flexible.